share/picongpu/tests/CurrentDeposition/bin/ci.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+#
+# This file is part of PIConGPU.
+# Copyright 2023 PIConGPU contributors
+# Authors: Mika Soren Voss, Hannes Wolf, Klaus Steiniger
+# License: GPLv3+
+#
+
+function absolute_path()
+{
+    builtin cd -- $1 && pwd
+}
+
+help()
+{
+  echo "Execute a single particle simulation over one time step and validate the current density values on the grid."
+  echo "Simulation results are validated against a minimal python implementation of current deposition on a grid."
+  echo ""
+  echo "Usage: ci.sh"
+  echo ""
+  echo "Options"
+  echo "-t | --flag                   - flag number of cmakeFlags"
+  echo "-h | --help                   - show help"
+  echo ""
+}
+
+## not used at the moment
+currentPath=$(cd `dirname $0` && pwd)
+currentPath=$(absolute_path $currentPath)
+
+
+#####################
+## option handling ##
+#####################
+# options may be followed by
+# - one colon to indicate they have a required argument
+OPTS=`getopt -ot:h help flag: -- "$@"`
+if [ $? != 0 ] ; then
+    # something went wrong, getopt will put out an error message for us
+    exit 1
+fi
+
+eval set -- "$OPTS"
+
+# parser
+while true ; do
+    case "$1" in
+    -t|--flag)
+            flag=$2
```

Comment:
if the user does not start the script with `-t`, you have to use `flag=0`

Reply:
thanks, that works.